.github/workflows/semgrep.yml

```diff
@@ -16,7 +16,21 @@ jobs:
       SEMGREP_APP_TOKEN: ${{ secrets.SEMGREP_APP_TOKEN }}
     container:
       image: semgrep/semgrep
+        
     if: (github.actor != 'dependabot[bot]')
     steps:
-      - uses: actions/checkout@v4
-      - run: semgrep ci
+      - uses: actions/checkout@v4    
+      - name: Checkout semgrep-utilities repo
+        uses: actions/checkout@v4
+        with:
+          repository: zeta-chain/semgrep-utilities
+          path: semgrep-utilities
+          
+      - run: semgrep ci --json --output semgrep-findings.json 
```

Comment:
`semgrep` supposedly has a `--sarif-output` flag, wouldn't that work?

ref: https://semgrep.dev/docs/cli-reference

Reply:
Ok can you add a comment about this?